cmd/contact-exporter/main.go

```diff
@@ -6,14 +6,13 @@ import (
 	"fmt"
 	"io/ioutil"
 	"os"
-	"sort"
-	"strings"
 
 	"gopkg.in/gorp.v1"
 
 	"github.com/jmhodges/clock"
 	"github.com/letsencrypt/boulder/cmd"
 	blog "github.com/letsencrypt/boulder/log"
+	"github.com/letsencrypt/boulder/mail"
```

Comment:
Could you import this as `bmail`, even though we aren't importing the stdlib version it makes reading the code a little less confusing.